<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWindow</class>
 <widget class="QMainWindow" name="ReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>115</y>
      <width>231</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <layout class="QVBoxLayout" name="Unique">
     <item>
      <widget class="QLabel" name="lb_0">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_1">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_2">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_3">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_4">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_5">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_6">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_7">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_9">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_10">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_11">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_12">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>D0X: No-gest: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_13">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>G09: 2click-2f: 12</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_16">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>G09: 2click-2f: 12</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_15">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>   Total:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>10</y>
      <width>221</width>
      <height>781</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="Rads_1">
     <item>
      <widget class="QRadioButton" name="rb_0">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_1">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_9">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_10">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_11">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_12">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_13">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_14">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_15">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_16">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_17">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_18">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_19">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_20">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_21">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_22">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>10</y>
      <width>221</width>
      <height>781</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <layout class="QVBoxLayout" name="Rads_2">
     <item>
      <widget class="QRadioButton" name="rb_23">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_24">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_25">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_26">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_27">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_28">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_29">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_30">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_31">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_32">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_33">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_34">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_35">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_36">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_37">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_38">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_39">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_40">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_41">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_42">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_43">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_44">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_45">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[00] D0X: 0001 - 0032</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>723</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Show Frame [00]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>73</y>
      <width>181</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Video Report:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_t">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>251</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1CM1_1_R_#217</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
